src/Flysystem/FlysystemAssetStore.php

```diff
@@ -291,7 +289,7 @@ function (ParsedFileID $parsedFileID, Filesystem $fs) {
      * @param bool $strictHashCheck
      * @return mixed
      */
-    private function applyToFileOnFilesystem(callable $callable, ParsedFileID $parsedFileID, $strictHashCheck = true)
+    protected function applyToFileOnFilesystem(callable $callable, ParsedFileID $parsedFileID, $strictHashCheck = true)
```

Comment:
`applyToFileOnFilesystem` and `applyToFileIDOnFilesystem` are the de facto replacement for a few methods.

It makes sense to add official support for them now by making them protected and allowing sub classes to access them directly.